### DISCUSSION TITLE
Issue with built-in icons not showing up after building and starting the Next.js project

### DISCUSSION THREAD

**Discussion by Ascurse** [VKCOM/VKUI#5305] [Q&A] [answered]
Hi there,

I'm encountering an issue with the VKUI library in my Next.js project. Specifically, I'm using Next.js with TypeScript and facing a problem with the built-in icons not displaying correctly after building and starting the project using "npm run build" and "npm run start".

When I'm running the project on the development server with "npm run dev", the icons appear as expected. However, after building the project and starting it using "npm run build" and "npm run start", the icons are no longer visible.

I have checked that the icons are imported correctly and that the necessary dependencies are installed.

Could you please help me understand why the icons are not showing up after building and starting the Next.js project? Is there anything specific that needs to be done to ensure the icons are included correctly in the build?

Here are some details about my setup:

Next.js version: 13.4.1
TypeScript version: 5.0.4
"@vkontakte/icons": "^2.43.0",
"@vkontakte/vk-bridge": "^2.7.4",
"@vkontakte/vkui": "^5.5.2",
I appreciate any assistance you can provide in resolving this issue. Thank you in advance!

**Comment by SevereCloud** [comment]
Hi, can you send an example?

https://codesandbox.io/p/sandbox/vkui-next-js-cssm-forked-p6zthf

**Reply by Ascurse** [reply, marked as answer]
Apologies for the delay in responding. I've successfully resolved the problem.
The issue arose when utilizing the new NextJS App Router. However, once I reverted back to using pages, it started functioning as anticipated.
